steward/src/commands/keys/cosmos/delete.rs

```diff
@@ -3,22 +3,26 @@ use abscissa_core::{clap::Parser, Application, Command, Runnable};
 use signatory::FsKeyStore;
 use std::path::Path;
 
+/// Steward keys cosmos delete [name]
 #[derive(Command, Debug, Default, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Delete a Cosmos Key.\n This command deletes a Cosmos key from your keystore when provided with the keyname."
+)]
 pub struct DeleteCosmosKeyCmd {
-    pub args: Vec<String>,
+    /// Cosmos keyname in keystore, takes a String
+    pub name: String,
 }
 
-/// The `gork keys cosmos delete [name] ` subcommand: delete the given key
+/// The `steward keys cosmos delete [name] ` subcommand: delete the given key
 impl Runnable for DeleteCosmosKeyCmd {
     fn run(&self) {
         let config = APP.config();
         // Path where key is stored.
         let keystore = Path::new(&config.keystore);
         let keystore = signatory::FsKeyStore::create_or_open(keystore).unwrap();
-        // Collect key name from args.
-        let name = self.args.get(0).expect("name is required");
-        let name = name.parse().expect("Could not parse name");
-        // Delete keyname after locating file from path and key name.
+        // Collect keyname from name arguement.
```

Comment:
We can drop the comments in this `run` method, the code is straightforward enough.